pkg/agent/util/net_linux.go

```diff
@@ -262,6 +262,49 @@ func GetNetRoutesOnAdapter(linkIndex int) (string, []interface{}, error) {
 	return gw, routes, nil
 }
 
+func HostInterfaceExists(ifName string) bool {
```

Comment:
In net_linux.go, you only reorder the functions, but not modify any implementations, right? If so, maybe these modification should be addressed by @mengdie-song ? @Anandkumar26

Reply:
what modifications do you suggest for this function?

Reply:
yes, I just reordered the functions so that all the private functions are towards the end of the file

Reply:
I just want to confirm the changes in this file with you, and if it is not about logic change on the function implementation,  I think it should be addressed when first provide the functions as your dependent patch is not merged yet. Then we could reduce such unnecessary review effort.

Reply:
Agree.